lib/apartment/active_record/postgresql_adapter.rb

```diff
@@ -0,0 +1,16 @@
+module Apartment::PostgreSqlAdapterPatch
+  def default_sequence_name(table, _column)
+    res = super
+    schema_prefix = "#{Apartment::Tenant.current}."
+    if res.starts_with?(schema_prefix) && Apartment.excluded_models.none?{|m| m.constantize.table_name == table}
```

Comment:
I wonder if we should check `Apartment.use_schemas` here. Is that flag still supported?

Reply:
The flag is still in place and i don't recall having changed anything around this, so we can most likely rely on it